float64 x
float64 y
int64 grid_x
int64 grid_y
float64 data
float64 footprint_size